app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,16 @@
+<p>
+  <%= post.like.all.count %></br>
+  <% if post.like.all.where(user: current_user).length===0 %>
+    <%= link_to 'Like Post', new_blog_post_like_path(post.blog, post) %>
+  <% else %>
+    <%= link_to 'Unlike post', blog_post_like_path(post.blog, post), method: :delete %>
+  <% end %>
+  </br>
+  <%= post.user.try(:email) || "deleted_user" %></br>
```

Comment:
Out of curiosity, why put the user's email here? Just as some indication of which user made the post? 

(Obviously exposing a user's email on any kind of real world blogging platform would be a bad thing!)